src/test/groovy/com/athaydes/spockframework/report/internal/VividAstInspectorSpec.groovy

```diff
@@ -96,7 +96,7 @@ class VividAstInspectorSpec extends Specification {
         blocks[ 1 ].statements.isEmpty()
         blocks[ 2 ].statements.isEmpty()
         blocks[ 3 ].statements == [ 'x == y' ]
-        blocks[ 4 ].statements.isEmpty() // where statements are not captured
+        blocks[ 4 ].statements == [ "x | y", "'a' | 'a'", "'b' | 'c'" ]
```

Comment:
So, the lines with the examples do not get captured, right? Wasn't that what you wanted to have?

Reply:
Ah! Sorry, looked too fast... those `a` and `b`s ARE THE EXAMPLES :D!?

Reply:
Yes this is exactly what I was describing in the issue.
The entire `where` source code block is available as a list of statements exactly like 
all the other code blocks. 🥳 

Although, now that I look at this again: 
Correct me if I am wrong, but could it be that there is an indent missing in the header of the table?
Should `"x | y"` not be `"  x | y"`?
Because otherwise the `|` lines might not match? 🤔